taskcluster/docker/node/build.py

```diff
@@ -265,7 +265,30 @@ def main():
         "artifacts": [],
     }
 
-    if os.environ.get("XPI_INSTALL_TYPE", "yarn") == "yarn":
+    install_type = os.environ.get("XPI_INSTALL_TYPE")
+    if install_type == "mach":
+        vcs_path = os.environ["VCS_PATH"]
+        # Keep objdir outside of topsrcdir so it isn't part of the checkout cache
+        objdir = "/builds/worker/objdir"
+        env = os.environ.copy()
+        env.update({
+            "MOZCONFIG": "browser/config/mozconfigs/linux64/browser-extensions",
+            "MOZBUILD_STATE_PATH": "/builds/worker/.mozbuild",
```

Comment:
Would it be worth caching the mozbuild state dir?

Reply:
I don't think so? Building these extensions doesn't require any toolchains or anything like that, so I think it would only really be saving the initial setup of this directory.